src/main/scala/de/uhh/lt/jst/ElasticSearchSparkJob.scala

```diff
@@ -8,32 +8,38 @@ abstract class ElasticSearchSparkJob extends SparkJob {
 
 
   abstract class ElasticSearchParser extends Parser {
+
+    val defaultUser = "elasticsearch"
```

Comment:
I am really cannot understand this part. The defaults should be part of the Config case class, not simple variables? Was not it the entire idea? I am really puzzled about it...

Reply:
I am puzzled how to answer it. I need a more specific critic?

Reply:
It would be technically impossible to add these default values to any Config case class as ElasticSearchSparkJob only inherits the type member ConfigType from the Job base class, but no concrete class.

Reply:
ok. i see now. that's no problem. thanks!